Slack.NetStandard/WebApi/IReactionsApi.cs

```diff
@@ -7,11 +7,8 @@ public interface IReactionsApi
     {
         Task<WebApiResponse> Add(string channel, Timestamp timestamp, string name);
         Task<ReactionGetResponse> Get(string channel, Timestamp timestamp, bool? full = null);
-
-        Task<ReactionListResponse> List(string user);
```

Comment:
Would you be able to move this to a ReactionListRequest object in the same way you did with the IUsersApi - moving to a request object model for those calls with Cursor + Limit options makes more sense moving forward, to more easily accommodate future changes.

Reply:
Done!